extensions/ql-vscode/src/vscode-tests/no-workspace/helpers.test.ts

```diff
@@ -225,4 +227,81 @@ describe('helpers', () => {
       message: 'My prefix (Size unknown)',
     });
   });
+
+  describe('open dialog', () => {
+    let showInformationMessageSpy: Sinon.SinonStub;
+    beforeEach(() => {
+      showInformationMessageSpy = sandbox.stub(window, 'showInformationMessage');
+    });
+
+    it('should show a binary choice dialog and return `yes`', (done) => {
+      // pretend user clicks on the url twice and then 'yes'
```

Comment:
This comment doesn't match the `onCall(0)` - here you're just testing a direct `yes` click right?